packs/data/actions.db/mark-for-death.json

```diff
@@ -16,7 +16,146 @@
         "requirements": {
             "value": ""
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.MarkForDeath.TargetMark",
```

Comment:
You can leave the label out. If there's no label, it defaults to the name of the source item.

If you are adding a specific label, you also need to update re-en.json to specify what will be displayed.

Reply:
Added the labels in the new commit.